napari/layers/utils/text.py

```diff
@@ -0,0 +1,192 @@
+from typing import Union
+
+import numpy as np
+
+from ._text_constants import TextMode
+from .text_utils import format_text_direct, format_text_properties
+from ...utils.colormaps.standardize_color import transform_color
+from ...utils.event import EmitterGroup, Event
+
+
+class TextManager:
```

Comment:
This is looking good - I'm thinking now about how much simpler this class will become if we go with the approach in #1436!

Reply:
> I'm thinking now about how much simpler this class will become if we go with the approach in #1436!

For sure. I think the majority of the typing I did for `TextManager` was adding the property getters/setters.

Reply:
Done!